site/layouts/partials/docs-navbar.html

```diff
@@ -34,12 +34,12 @@
           <a class="nav-link{{ if eq .Page.Title "Boostwatch" }} active{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/boostwatch/" {{ if eq .Page.Title "Boostwatch" }}aria-current="page"{{ end }} onclick="dataLayer.push({'event': 'clic', 'site_name':'accessibility-boosted', 'phase':'prod', 'track_category':'navbar', 'track_name':'boostwatch'});">Boostwatch</a>
         </li>
       </ul>
-      <ul class="navbar-nav flex-row ml-md-auto d-none d-md-flex">
```

Comment:
It is now possible to select the version of the docs under md breakpoints. Dropdown is a bit ugly but at least this feature is now accessible.

Reply:
This is a great feature ! 

However, I have 2 points here: 
- the version button is over the dropdown border
- missing border between last item and version button/github button
![image](https://user-images.githubusercontent.com/91960143/175546647-19ef4227-fc7e-43ca-b1b4-71d60855e7af.png)

Reply:
Fixed some details via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1304/commits/2f7630191dd34b4c0340f78c37ac167b462008bc